python/pyspark/sql/tests.py

```diff
@@ -2367,6 +2380,151 @@ def range_frame_match():
 
         importlib.reload(window)
 
+
+class TypesTest(unittest.TestCase):
+
+    def test_verify_type_ok_nullable(self):
+        for obj, data_type in [
+                (None, IntegerType()),
+                (None, FloatType()),
+                (None, StringType()),
+                (None, StructType([]))]:
+            _verify_type(obj, data_type, nullable=True)
+            msg = "_verify_type(%s, %s, nullable=True)" % (obj, data_type)
+            self.assertTrue(True, msg)
```

Comment:
I think we should surround `_verify_type(obj, data_type, nullable=True)` with try block and check if it raises an exception or not as the same as we do in `test_verify_type_not_nullable` test.